packages/overlay/src/integrations/sentry/index.ts

```diff
@@ -18,98 +21,108 @@ type SentryIntegrationOptions = {
 };
 
 export default function sentryIntegration(options?: SentryIntegrationOptions) {
-  return {
-    name: 'sentry',
-    forwardedContentType: [HEADER],
-
-    setup: ({ open }) => {
-      addSpotlightIntegrationToSentry(options);
-
-      const spotlightEventTarget = getSpotlightEventTarget();
-
-      const onRenderError = (e: CustomEvent) => {
-        log('Sentry Event', e.detail.event_id);
-        if (e.detail.event) sentryDataCache.pushEvent(e.detail.event);
-        // TODO: handle async
-        open(`/errors/${e.detail.eventId}`);
-      };
-
-      spotlightEventTarget.addEventListener('sentry:showError', onRenderError as EventListener);
-
-      return () => {
-        spotlightEventTarget.removeEventListener('sentry:showError', onRenderError as EventListener);
-      };
-    },
-
-    processEvent: (event: RawEventContext) => processEnvelope(event),
-
-    tabs: () => {
-      const errorsCount = sentryDataCache
-        .getEvents()
-        .filter(
-          e =>
-            e.type != 'transaction' &&
-            (e.contexts?.trace?.trace_id ? sentryDataCache.isTraceLocal(e.contexts?.trace?.trace_id) : null) !== false,
-        ).length;
-
-      const localTraces = sentryDataCache.getTraces().filter(t => sentryDataCache.isTraceLocal(t.trace_id) !== false);
-
-      return [
-        {
-          id: 'errors',
-          title: 'Errors',
-          notificationCount: {
-            count: errorsCount,
-            severe: errorsCount > 0,
+  if (checkBrowserSDKVersion()) {
```

Comment:
Can we extract the two integrations here so that not the entire code is indented by `checkBrowserSDKVersion`?

Reply:
So basically, do something like:

```
if (checkBrowserSDKVersion()) {
  return _sentryIntegration();
} else {
  return _sdkNotCompatibleSentryIntegration()
}
```

No particular strong feelings about naming or details just wanna keep this somewhat readable :)